frontend/src/pages/settings/account/Account.tsx

```diff
@@ -38,6 +57,24 @@ function AccountPage() {
             <Grid item md={4}>
               <PasswordForm />
             </Grid>
+            <Grid item>
+              <Box marginBottom={2} marginTop={2}>
+                <Typography variant="h4" color="secondary">
+                  Delete account
+                </Typography>
+                <Divider />
+              </Box>
+              <Grid item md={4}>
+                <Button
+                  className={classes.warning}
+                  onClick={deleteAccount}
+                  fullWidth
+                  variant="contained"
+                >
+                  Delete your account
+                </Button>
+              </Grid>
+            </Grid>
```

Comment:
I would find it better to make this into a separated component like for the `PasswordForm` above.

Maybe even add a simple unit test for it (you may ask @GresilleSiffle for support on this)